src/main/java/seedu/patientist/logic/commands/UpdateWardCommand.java

```diff
@@ -12,13 +12,15 @@
 import seedu.patientist.model.Patientist;
 import seedu.patientist.model.person.Person;
 import seedu.patientist.model.person.patient.Patient;
+import seedu.patientist.model.person.staff.IsStaffPredicate;
+import seedu.patientist.model.person.staff.Staff;
 import seedu.patientist.model.ward.Ward;
 
 /**
  * Update Patient's ward identified using it's displayed index from the patientist book.
  */
 
-public class UpdatePatientWardCommand extends Command {
+public class UpdateWardCommand extends Command {
     public static final String COMMAND_WORD = "trfward";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Moves the patient."
```

Comment:
Change this to `Transfers the person indicated by the index to the ward specified. `. Remember to add a space after the fullstop.